i18n/readme.md

```diff
@@ -0,0 +1,20 @@
+# About en-US
+
+**TL;DR;** Do not manually modify the contents of `i18n/en-US`.
+
+Docusaurus translations for anything that is not markdown are stored
+in several JSON files. These files are generated from the
+contents of `docusaurus.config.js`, `sidebars.js`, and installed
+plugins.
+Some of the files that are generated are `code.json`, `navbar.json`,
+`footer.json`, etc.
+
+Because those files only exist in a "translation" and not the "default"
+site, we create them in the `en-US` folder via
+`yarn write-translations --locale en-US` to uploaded them to crowdin via
+`electron/i18n` and then downloaded for the right locale.
```

Comment:
suggestion: reworded this paragraph  to make things clearer and added relevant links

```suggestion
For the website's source language (`en-US`), these JSON files are **not**
used by Docusaurus. Rather, they serve as source files for our
[Crowdin](https://crowdin.com/project/electron) internationalization (i18n)
workflow. The [`electron/i18n`](https://github.com/electron/i18n) repository
periodically updates these source strings on Crowdin via
[GitHub Actions](https://github.com/electron/i18n/actions/workflows/schedule-update-source-content.yml).
```